lib/teleterm/gatewaytest/helpers.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gatewaytest
+
+import (
+	"fmt"
+	"net"
+	"testing"
+	"time"
+
+	apiutils "github.com/gravitational/teleport/api/utils"
+
+	"github.com/gravitational/trace"
+	"github.com/stretchr/testify/require"
+)
+
+// BlockUntilGatewayAcceptsConnections attempts to initiate a connection to the gateway on the given
+// address. It will time out if that address doesn't respond after 1 second.
+func BlockUntilGatewayAcceptsConnections(t *testing.T, address string) {
+	conn, err := net.DialTimeout("tcp", address, time.Second*1)
```

Comment:
I am surprised 1 second is enough for our CI builds =p

Reply:
Dang, you really think so? I increased them to 5 seconds just to be safe. ;)

Reply:
Seems it was using 1 second already but not one complains about these tests. =D